GetFed/GetFed/Helpers/APIClient.swift

```diff
@@ -10,10 +10,28 @@ import Foundation
 import Alamofire
 import CoreData
 
-class APIClient {
+final class APIClient {
+    
+    let appId = EdamamAppID
+    let appKey = EdamamAppKey
+    static let shared = APIClient()
+    
+    func setURL(with searchText: String) -> URL? {
+        var urlComponents = URLComponents()
```

Comment:
Another thing to think about is using Alamofire's `URLRequestConvertible` and, in Alamofire 5, `Encodable` parameters that can be converted into a `URLRequest` with a query string. But this is a good start towards formalizing your URL generation.